conf/multiconfig/README.txt

```diff
@@ -0,0 +1,35 @@
+How to to set a multiconfig environment for ledge.
```

Comment:
it will be good to update setup-environment-internal  with next steps to build also.